templates/index.html

```diff
@@ -138,27 +138,38 @@ <h6><span id="displayStatsTitle">Which states feel the strongest about each genr
 
 				<div class="row">
 					<div id="mapCanvas">
-						<div id = "countymap"></div>
+						<div id="countymap"></div>
 						<!-- div and svg to hold the map. -->
 						<svg id="statesvg" viewBox="0 0 960 630" preserveAspectRatio="xMidYMid meet"></svg>
 						<div id="tooltip"></div><!-- div to hold tooltip. -->
 
 					</div>
 </body>
 
-
-<script src="{{url_for('static',filename = 'libs/d3.min.js')}}"></script><!-- D3 graphing library -->
+<!-- D3 graphing libraries -->
+<script src="{{url_for('static',filename = 'libs/d3.min.js')}}"></script>
 <script src="{{url_for('static',filename = 'libs/d3-scale-chromatic.v1.min.js')}}"></script>
-<script src="{{url_for('static',filename = 'libs/topojson.v1.min.js')}}"></script><!-- topojson library -->
+<!-- topojson library -->
+<script src="{{url_for('static',filename = 'libs/topojson.v1.min.js')}}"></script>
+
+<!-- leaflet libraries -->
 <script src="{{url_for('static',filename = 'libs/leaflet/dist/leaflet.js')}}"></script>
 <script type="text/javascript" src="http://maps.stamen.com/js/tile.stamen.js?v1.3.0"></script>
 <script src="{{url_for('static',filename = 'libs/leaflet-choropleth/dist/choropleth.js')}}"></script>
+
+<!-- global variables and helper functions -->
 <script src="{{url_for('static',filename = 'js/globals.js')}}"></script>
 <script src="{{url_for('static',filename = 'js/helpers.js')}}"></script>
-<script src="{{url_for('static',filename = 'data/states_geo.js')}}"></script>
-<script src="{{url_for('static',filename = 'data/counties_geo.js')}}"></script>
+
+<!-- external data -->
 <script src="{{url_for('static',filename = 'data/countyData.js')}}"></script><!-- temporary data file -->
+<script src="{{url_for('static',filename = 'data/venue_points.js')}}"></script><!-- temporary data file -->
```

Comment:
You might have forgotten to push this file in the pull request?

Reply:
Oops. Just pushed it.